src/bundles/page-template.less

```diff
@@ -1,7 +1,7 @@
 @page-template-header-content-min-height: 32px;
-@page-template-header-z-index: 1;
-@page-template-content-z-index: 0;
-@page-template-footer-z-index: 0;
+@page-template-header-z-index: @z-index-cover-layer;
+@page-template-content-z-index: @z-index-base-layer;
+@page-template-footer-z-index: @z-index-base-layer;
```

Comment:
We could clean up the other z-indexes in this file (e.g.  'z-index: 1000;') and the maybe just inline the z-index variables since they are ach used just once.

Reply:
Yes, changed.